flash.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+#This Should work correctly, if not let me know -Sam
+os.system("avrdude -c usbasp -p t84 -B 20 -U flash:w:target/avr-attiny84/release/semasher.elf")
```

Comment:
I'd also like the output of the programmer to show up in the console. This currently just returns a posix style success code.